src/test/java/uk/gov/justice/hmpps/prison/web/config/CacheConfigIntTest.kt

```diff
@@ -67,7 +67,7 @@ class CacheConfigIntTest {
   fun `test that each cache is tested by this class`() {
     val nestedTests = this::class.nestedClasses.map { it.simpleName!!.replace("_.*".toRegex(), "") }.toSet()
 
-    assertThat(cacheManager.cacheNames.toSet().minus(nestedTests)).isEmpty()
+    assertThat(cacheManager.cacheNames.toSet().minus(nestedTests).minus("jwks")).isEmpty()
```

Comment:
don't need a test for the jwks as it is used internally and not for caching service requests